src/vivarium/framework/population/manager.py

```diff
@@ -415,10 +439,10 @@ def get_population(
             If any of the requested attributes do not exist in the population table.
         """
 
-        if self.population is None:
+        if self._private_columns is None:
```

Comment:
Should we add a constraint on `get_population()` to restrict during `setup()` and `post_setup()` (when we know it won't have been set yet)?